tools/rna_tools/rnasnp/rnasnp.xml

```diff
@@ -13,7 +13,8 @@
     <version_command>RNAsnp --version</version_command>
     <command>
 <![CDATA[
-
+    
+    export RNASNPPATH=`dirname \$(dirname \$(which RNAsnp))` &&
```

Comment:
@TorHou it didn't work! even though it worked locally
+Error: The parameter file couldn't be accessible at the path /tmp/tmp3l19F9/job_working_directory/000/3/conda-env/lib/distParam/W200parameters.bin